com.serveftp.officefactoring.yaml

```diff
@@ -0,0 +1,45 @@
+app-id: com.serveftp.officefactoring
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+rename-icon: officefactoring
+command: OfficeFactoring
+cleanup:
+  - /include
+  - /lib/pkgconfig
+  - /share/pkgconfig
+  - /share/aclocal
+  - /man
+  - /share/man
+  - /share/gtk-doc
+  - '*.la'
+  - '*.a'
+  - /lib/girepository-1.0
+  - /share/doc
+  - /share/gir-1.0
+finish-args:
+  - '--socket=x11'
+  - '--share=ipc'
+  - '--share=network'
+  - '--filesystem=host'
+  - '--metadata=X-DConf=migrate-path=/org/gnome/dictionary/'
```

Comment:
This feel like you did cut & paste from somewhere. It doesn't feel right

Reply:
Como que devo fazer nesse caso?